### PR TITLE
feat: allow fullscreen without alternative screen

### PR DESCRIPTION
**Pull request by rbrugo**
This should solve #766

### CODE REVIEW

File: include/ftxui/component/screen_interactive.hpp [hunk 1]
```diff
@@ -32,7 +32,7 @@ class ScreenInteractive : public Screen {
   static ScreenInteractive FixedSize(int dimx, int dimy);
   static ScreenInteractive Fullscreen();
```
Comment (ArthurSonzogni) [inline include/ftxui/component/screen_interactive.hpp:33]: What about:
```cpp
static ScreenInteractive FullscreenPrimaryScreen();
```
?

File: include/ftxui/component/screen_interactive.hpp [hunk 2]
```diff
@@ -32,7 +32,7 @@ class ScreenInteractive : public Screen {
...
   static ScreenInteractive FitComponent();
-  static ScreenInteractive TerminalOutput();
+  static ScreenInteractive TerminalOutput(bool use_alternative_screen = true);
```
Comment (ArthurSonzogni) [inline include/ftxui/component/screen_interactive.hpp:35]: note: You modified the wrong function.

**Comment by ArthurSonzogni** [comment]
Thanks!

**Comment by rbrugo** [comment]
I'm sorry for the late reply, now the issues should be fixed!

**Comment by ArthurSonzogni** [comment]
Thanks I merged the PR!

**Pull request closed without merging**